### ISSUE TITLE
datetime_parser.py strptime function format error

### ISSUE THREAD

**Issue by nubjub** [jdf76/plugin.video.youtube#809]
I'm getting something like: 
`Error Contents: time data u'2020-04-29T17:49:03Z' does not match format '%Y-%m-%dT%H:%M:%S.%fZ'` 
I've patched my installed version of `/resources/lib/youtube_plugin/kodion/utils/datetime_parser.py`to:
```python
def strptime(s, fmt="%Y-%m-%dT%H:%M:%S"):
    # noinspection PyUnresolvedReferences
    import _strptime
    try:
        stest = time.strptime(s, fmt)
    except ValueError as v:
        #trim anything after the seconds
        if len(v.args) > 0 and v.args[0].startswith('unconverted data remains: '):
            s = s[:-(len(v.args[0]) - 26)]
            r = datetime(*time.strptime(s, fmt)[:6])
        else:
            raise
    return r
```

**Comment by anxdpanic** [comment]
this is already fixed in 6.8.0 which is available on all the repositories.

**Comment by probonopd** [comment]
Ran into this too:

```
2020-05-01 12:41:10.125 T:3457934192  NOTICE: [plugin.video.youtube] Running: YouTube (6.7.0) on Leia (Kodi-18.2) with Python 2.7.15
                                            	Path: /special/new_uploaded_videos_tv/
                                            	Params: {}
2020-05-01 12:41:14.655 T:3457934192   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.ValueError'>
                                            Error Contents: time data u'2020-04-30T19:15:59Z' does not match format '%Y-%m-%dT%H:%M:%S.%fZ'
```

Upgrading to 6.8.0 indeed fixed this for me.
